dsc/resources/authoringResourceMOF.md

```diff
@@ -287,3 +287,16 @@ if (PsDscContext.RunAsUser) {
     Write-Verbose "User: $PsDscContext.RunAsUser";
 }
 ```
+
+## Rebooting the Node
+
+If the actions taken in your `Set-TargetResource` function require a reboot, you can use a global flag to tell the LCM to reboot the Node. This reboot occurs directly after the `Set-TargetResource` function completes.
+
+Inside your `Set-TargetResource` function, add the following line of code.
+
+```powershell
+Include this line if the resource requires a system reboot.
```

Comment:
Is this line supposed to be a comment or instructions? Comment it out or move it outside of the code block.

Reply:
Good catch, simple typo.